tests/admin_views/tests.py

```diff
@@ -887,6 +888,13 @@ def test_filter_with_custom_template(self):
         response = self.client.get("/test_admin/admin/admin_views/color2/")
         self.assertTemplateUsed(response, 'custom_filter_template.html')
 
+    def test_get_admin_log_template_tag(self):
+        """
+        Ensure that the 'get_admin_log' template tag works.
```

Comment:
Done. Also moved test to what looks like a better place next to another test testing the same tag.
Now at admin_changelist.tests.AdminLogNodeTestCase